config/clients/dotnet/template/README_initializing.mustache

```diff
@@ -10,7 +10,7 @@ namespace Example {
             try {
                 var configuration = new Configuration(storeId, environment) {
                     ApiScheme = Environment.GetEnvironmentVariable("{{appUpperCaseName}}_API_SCHEME"), // optional, defaults to "https"
-                    ApiHost = Environment.GetEnvironmentVariable("{{appUpperCaseName}}_API_HOST"), // required, define without the scheme (e.g. api.openfga.example instead of https://api.openfga.example)
+                    ApiHost = Environment.GetEnvironmentVariable("{{appUpperCaseName}}_API_HOST"), // required, define without the scheme (e.g. api.fga.example instead of https://api.fga.example)
```

Comment:
Why not openfga?

Reply:
To make it consistent. We use `api.fga.example` in other places like the tests

Reply:
since api.fga.example are used in multiple places, should we define them in common/config.base.json and reference it here?